Update to normalize.css v4.0.0

marktopper opened the pull request:
#19575

marktopper committed:
Update to normalize.css v4.0.0

#19575

marktopper commented on scss/_normalize.scss:
```diff
@@ -90,41 +93,49 @@ a {
-  border-bottom: 1px dotted;
+  border-bottom: none; // 1
+  text-decoration: underline; // 2
+  text-decoration: underline dotted; // 2
```
This is what the original normalize looks like as well.

https://github.com/necolas/normalize.css/blob/master/normalize.css#L113

marktopper committed:
Append HoundCi fixes

https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850395
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850397
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850398
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850401
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850399
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850402
https://github.com/twbs/bootstrap/pull/19639#discussion-diff-57850403

marktopper commented:
What is this last violation found by @houndci-bot?

cvrebert commented:
Please ignore HoundCI in this case and revert the "fixes" it wanted.
normalize.css is a special case since it's vendored.

marktopper committed:
Reverted

Reverted to original content from https://raw.githubusercontent.com/necolas/normalize.css/d311749e127ea3ea13eb900ad953270d62734f4f/normalize.css

marktopper commented:
@cvrebert Reverted to be exactly as the [vendor file](https://github.com/necolas/normalize.css/blob/d311749e127ea3ea13eb900ad953270d62734f4f/normalize.css).

cvrebert commented:
The `//` comment style is the one change that is desirable, because other than the initial attribution comment, [we don't want any comments in the compiled CSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments). So please use `//` comments like in the old version of the file.

marktopper committed:
Fix comment style

Fix comment style to `//`, because other than the initial attribution comment, [we don't want any comments in the compiled CSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments).

marktopper commented:
@cvrebert Comment style fixed for anything else than the first line.

mdo commented:
Superseded by #19862 to address changes needed on our end. Thanks @marktopper for getting us started! Your commits are part of that PR <3.
